docker-compose.yaml

```diff
@@ -18,7 +18,6 @@ services:
     extends:
       file: frontend/docker-compose-base.yaml
       service: frontend
-    image: ghcr.io/misarch/frontend:main
```

Comment:
?

Reply:
This must be a copy/paste error

Reply:
No idea how that ended up in there

Reply:
Need to be more careful what changes i make while debugging